### ISSUE TITLE
Where to add custom script so it is available to me via vagrant ssh

### ISSUE THREAD

**Issue by yoxalld** [Varying-Vagrant-Vagrants/VVV#858]
Hi,

I have a small bash script that allows me to install my starter theme and some of the basic plugins that I use on every project. However, because the script uses wp-cli I need to run it from within the vagrant box(via vagrant ssh) at least I think I do. So my question is where should I place that file within the vagrant setup so that a) I can alias from within vagrant ssh. b) It won't be overwritten with a vagrant halt/up

Thanks!

**Comment by aaroncampbell** [comment]
There are several synced folders (meaning they're available both in and out of the vagrant image):
`www`
`database`
`config`
`log`

Personally, I'd probably put it in the root of www, but you could even add your own synced folder if you want (but it's probably easier to just drop it at the root of www).

**Comment by yoxalld** [comment]
Hey @aaroncampbell thanks for the info! I did some more digging and I found a folder: www/phpcs/scripts that was already included in the $PATH for the vagrant ssh so I was able to drop my script in there so now I can call it from anywhere within my vagrant ssh session. Now I'm working through some edits because it was looking for folders on my local machine which now don't make sense because it's looking on the vagrant box instead.
